_docs/2.0.0/usage/run.md

```diff
@@ -67,6 +68,28 @@ For description of requirements and deployment see the [README.md][project-readm
 --report-version <data_run_version>
 ```
 
+## Running Standardization And Conformance together
```

Comment:
```suggestion
## Running Standardization and Conformance together
```

Reply:
I would perhaps keep it with capitalized _And_, as kind of semi-official name of the job.
But would add
```suggestion
## Running Standardization And Conformance together as one Spark job
```